firmware/application/apps/analog_tv_app.cpp

```diff
@@ -103,20 +94,14 @@ AnalogTvView::AnalogTvView(
     };
 
     tv.on_select = [this](int32_t offset) {
-        field_frequency.set_value(receiver_model.tuning_frequency() + offset);
+        field_frequency.set_value(receiver_model.target_frequency() + offset);
```

Comment:
Offset? Ensure this isn't trying to do a tuning offset.

Reply:
@jLynx I know you use that app, is there an offset anywhere?

Reply:
Seems like dead code. Nothing in ui_tv actually calls that.
I'll just leave it for now.